packages/axway-cli-auth/src/lib/keypair.js

```diff
@@ -0,0 +1,104 @@
+import path from 'path';
+import { existsSync, writeFileSync } from 'appcd-fs';
+import { initSDK } from '@axway/amplify-cli-utils';
+import { prompt } from 'enquirer';
+
+/**
+ * Generates a public/private keypair. It prompts for the output filenames and whether to overwrite
+ * if needed.
+ *
+ * @param {Object} opts - Various options.
+ * @param {Boolean} [opts.force] - When `true`, uses the default output filenames and overwrites if
+ * already exists.
+ * @param {String} [opts.privateKey] - The path to write the generated private key to.
```

Comment:
Is it also allowing to provide the file name?

Reply:
Yup! `opts.privateKey` and `opts.publicKey` are paths to the respective files.